frontend/app/src/i18n/en.json

```diff
@@ -108,6 +108,25 @@
     "accountCopiedToClipboard": "Account address copied to clipboard!",
     "accounts": "Accounts",
     "accountSuspended": "Account suspended",
+    "accountLinkingCode": {
```

Comment:
You'll need to run `npm run sync_lang` from `/frontend/app` to generate the other languages.